### ISSUE TITLE
ATL on mask displayable of AlphaMask() does not reset with Rollback

### ISSUE THREAD

**Issue by juniorthejunior** [renpy/renpy#5894]
If you have some ATL animation on the mask displayable of `AlphaMask()`, the animation plays correctly the first time that it is shown, but if you rollback and progress again (not just with rollforward, but also regular progressing), the animation is now always stuck at its final step - in other words, the ATL animation given to the mask displayable does not restart with rollback.

The exact same thing happens with the `image` displayable of `Movie()`: when `image` is finally shown after the video, it is always stuck at the last state of its ATL animation on subsequent replays. (I'm assuming these problems have the same root, so just mentioning this here as well.)

In both cases, `scene expression Movie(...)/AlphaMask(...)` gets around the issue.

Tested on 8.4.0.24110907+nightly
```py
transform my_tm():
    xalign .5 xsize 0
    linear 5.0 xsize 500

image alphamask_test = AlphaMask(
    "gui/window_icon.png",
    mask=At(Solid("#fff"), my_tm),

    align=(.5, .5),
)

label start:
    pause
    scene alphamask_test
    pause
    scene expression AlphaMask("gui/window_icon.png", At(Solid("#fff"), my_tm), align=(.5, .5))
    pause
```

**Comment by renpytom** [comment]
Fixed by fd027dbdbd74ad238c8c2f27cc712b8db74b542d .

**Comment by juniorthejunior** [comment]
Appreciate it Tom, should I open a separate Issue for `Movie()`?
